src/SprykerEco/Glue/UnzerRestApi/UnzerRestApiConfig.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Glue\UnzerRestApi;
+
+use Spryker\Glue\Kernel\AbstractBundleConfig;
+
+class UnzerRestApiConfig extends AbstractBundleConfig
+{
+    /**
+     * @api
+     *
+     * @var string
+     */
+    public const UNZER_NOTIFICATIONS = 'unzer-notification';
```

Comment:
not fixed

`RESOURCE_UNZER_NOTIFICATIONS = 'unzer-notifications';`

Reply:
Fixed.